packages/docs/docs/medications/formulary.md

```diff
@@ -0,0 +1,298 @@
+- The `intendedRoute` attribute suggests how the drug is to be ingested, whether orally, intravenously, etc. The [SNOMED Route Codes](http://hl7.org/fhir/R4/valueset-route-codes.html) are available for more information.
+
+We will be providing a comprehensive example of a [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource that incorporates drugCharacteristics, packaging, doseForm, and intendedRoute.
+
+:::note A note about units
+
+All units for medication quantities are [UCUM units of measure](https://terminology.hl7.org/4.0.0/ValueSet-v3-UnitsOfMeasureCaseSensitive.html). This includes standard SI units, as well as healthcare specific unit codes. A common unit for medications is `{tbl}`, which stands for "tablets", which is useful for quantifying the amount of medication in a package.
+
+:::
+
+<details>
+<summary>
+Example Drug Characteristics
+</summary>
+
+```ts
+{
+  "resourceType": "MedicationKnowledge",
+  "code": {
+    "text": "acetaminophen 325 MG [Tylenol]",
+    "coding": [
+      {
+        "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+        "code": "569998"
+      },
+      {
+        "system": "http://hl7.org/fhir/sid/ndc",
+        "code": "5058049501"
+      }
+    ]
+  },
+  "drugCharacteristic": [
+    {
+      "type": {
+        "text": "Color",
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-characteristic",
+            "code": "color"
+          }
+        ]
+      },
+      "valueString": "white"
+    },
+    {
+      "type": {
+        "text": "Imprint Code",
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-characteristic",
+            "code": "imprintcd"
+          }
+        ]
+      },
+      "valueString": "Tylenol"
+    }
+  ],
+  "amount": {
+    "value": 100,
+    "unit": "{tbl}"
+  },
+  "packaging": {
+    "type": {
+      "text": "Bottle",
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-package-type",
+          "code": "bot",
+          "display": "Bottle"
+        }
+      ]
+    },
+    "quantity": {
+      "value": 100,
+      "unit": "{tbl}"
+    }
+  },
+  "doseForm": {
+    "text": "tablet"
+  },
+  "intendedRoute": [
+    {
+      "text": "oral"
+    }
+  ]
+  //...
+}
+```
+
+</details>
+
+## Drug Classifications and Regulations
+
+The `MedicationKnowledge.productType` field can be used to categorize the drug within the formulary, potentially along multiple dimensions. This field can also be used to describe:
+
+- Whether a drug is generic or branded
+- Whether the drug is prescribable
+- Whether the drug is dispensable
```

Comment:
May want to provide a little context on what dispensable means in this context and an example.